diesel/src/expression/expression_methods/sort_expression_methods.rs

```diff
@@ -0,0 +1,44 @@
+use expression::predicates::*;
+
+#[cfg(feature = "postgres")]
+pub trait Ordering : Sized {
```

Comment:
FYI: Searching the docs for "trait:ord" shows that there is a `diesel::types::SqlOrd`.